### DISCUSSION TITLE
0.5.0

### DISCUSSION THREAD

**Discussion by z-huang** [z-huang/music#757] [Announcements]
- New app icon
- Rewrite UI using Jetpack Compose
- Better UI/UX
- Dynamic theme
- Redesigned home screen
- Improved download experience
- New lyrics source: YouTube subtitle
- Show replay button when playing ended

<hr /><em>This discussion was created from the release <a href='https://github.com/z-huang/InnerTune/releases/tag/v0.5.0'>0.5.0</a>.</em>

**Comment by 11xjas** [comment]
https://play.google.com/store/apps/details?id=com.zionhuang.music

Same package but Different developer?

**Reply by astarq** [reply]
Yep, Aurora Store offered me an update for that too. I don't understand what people get out of changing an app's branding and impersonating it!?
Regardless, congrats on the 5x release z-huang! Thanks for all the good work ❤️

**Reply by z-huang** [reply]
Yes. The developer shouldn't use the same package name as mine, and ze also violates GPLv3 by not releasing zir source code.

**Reply by tomc128** [reply]
Just left a 1 star review on the app. Hope you can get something sorted regarding taking it down!

**Reply by wolf-yuan-6115** [reply]
> Just left a 1 star review on the app. Hope you can get something sorted regarding taking it down!

I've already report it several months ago, nothing happened.

**Reply by mmtxrc** [reply]
![Screenshot_2023-07-12-11-56-05-067_org mozilla firefox](https://github.com/z-huang/InnerTune/assets/94820934/d5081038-9b75-428b-a401-ef951e6a15e8)

wow they crediting him ppl! they so noble oml
i find it so pointless. so friggin pointless. its scavenging for money. retards out there geez.

**Reply by wolf-yuan-6115** [reply]
> ![Screenshot_2023-07-12-11-56-05-067_org mozilla firefox](https://github.com/z-huang/InnerTune/assets/94820934/d5081038-9b75-428b-a401-ef951e6a15e8)
> 
> wow they crediting him ppl! they so noble oml
> i find it so pointless. so friggin pointless. its scavenging for money. retards out there geez.
> 

I don't know why it can pass google checking, they shouldn't allow this kind of app exist on Google play.

**Comment by Yonle** [comment]
Looks like number of function features (like automatically add song to library) was unavailable in this version,

Notably changes at the home contents too.

But overall things works well. Hope things goes better in the next version 👍🏻

**Comment by Rahulsourcecode** [comment]
Moods & genres function on the previous version was a good feature to have but in this version that's not there

**Reply by MohamedxSalah** [reply]
Agree, the old home tab was filled with a lot of helpful categories and sections

**Comment by MohamedxSalah** [comment]
Thanks for the update!
As pointed out by other users, some features are missing from this update
Also some animations like the play button animation which was a nice touch

Another subjective thing, is the setting large button in the home tab, i prefer the old 3 dots drop down menu as oppose to the big current one

Thanks a ton in advance 🙏 I really appreciate your work ❤️

**Reply by MJC579** [reply]
Disable dynamic theme under appearance in the settings menu.

**Comment by MohamedxSalah** [comment]
> Disable dynamic theme under appearance in the settings menu.

Oh wow , i feel dumb that i didn't notice that😂
Thanks a ton 🙏
I edited my comment now

**Comment by NeeRaj-2401** [comment]
Just discovered the double tap to seek and its surprisingly good, cause sometimes useful to me :)

**Comment by seasidesongs** [comment]
Loving the app
Just need the following tweaks
To be able to bookmark/like artists
Moods and genres back
The settings tab to be hidden/ smaller

**Comment by Sgt-Soap** [comment]
Looks great, but some features like "auto add songs to library" and "auto download songs in library" are no longer supported. Which were really great features in my opinion. Would like to see said features make a comeback.

**Comment by Zaxx69** [comment]
Bro the app is great, but their are some minor problem, some songs don't play while clicking upon and shows "unknown error" for example crystal(sped up)

**Reply by Zaxx69** [reply]
<img width=200 src="https://github.com/z-huang/InnerTune/assets/138523882/551e9733-63e6-4547-92f0-18f51a0cc9e1" />

**Reply by mmtxrc** [reply]
its likely when the app was left idle for a long time. happened with me too. i restarted and it worked fine.

**Reply by TheTrashCrafter** [reply]
Some age restricted songs don't work because of piped.

**Comment by thornySoap** [comment]
Is there a particular reason for not beeing able to customize the tabs anymore? It's useful to me alone because my screen is to narrow for five elements to be displayed with propriate padding to the edges. Also is it planned to bring back the navigation rail?

**Comment by ThatOneUnoriginal** [comment]
I am wondering when InnerTune 0.5.0 would be pushed to F-Droid? Used that to download InnerTune and right now it states the latest version of the application is 0.4.4

**Reply by z-huang** [reply]
#786

**Reply by z-huang** [reply]
It was pushed to F-Droid.

**Comment by JerryHong08** [comment]
where is the explore button?

**Reply by z-huang** [reply]
Redesigning in home screen. Tracking in #712.

**Comment by codewhizz** [comment]
Thank you! Best Music App.

**Comment by Osanosa** [comment]
Yep, y'know, this is good. But:
1) material colors switch is flipped
2) UI is not that intuitive, ytm is better in a mini player section, they made it clear of where to swipe where to get what you want, e.g. open queue, close it, minimize, etc.

Otherwise nice job, keep going!

**Reply by mmtxrc** [reply]
i disagree but sure

**Comment by Thevoid30** [comment]
Some way could be added for restoring songs will download automatically? I manage 2 devices and when sharing my playlist I must download one by one 

Other than that it's an amazing app 👍
